#!/usr/bin/python3
def raise_exception_msg(message=""):
    # Raising a NameError exception with a custom message
    raise NameError(message)
